mapper/map.lua

```diff
@@ -338,8 +338,7 @@ function amap:check_room_on_direction_of(room, dir, force)
     if not force and n_exits and n_exits["up"] and not n_exits[dir] then
         local to_check_x, to_check_y, to_check_z = getRoomCoordinates(n_exits["up"])
         to_check_y = -to_check_y
-        if amap:check_direction_coords_correctness(amap.curr.x, amap.curr.y, amap.curr.z, to_check_x, to_check_y, to_check_z, dir)
-                and not amap.long_to_short[k] then
```

Comment:
Nie wiem dokładnie jaka była intencja - ale w związku z tym, że nie ma zmiennej `k` to druga część warunku jest zawsze `true` - usuwam.